server/container_create.go

```diff
@@ -607,6 +611,11 @@ func (s *Server) CreateContainer(ctx context.Context, req *pb.CreateContainerReq
 
 	newContainer.SetCreated()
 
+	if ctx.Err() == context.Canceled || ctx.Err() == context.DeadlineExceeded {
+		log.Infof(ctx, "ctrCreate: context was either canceled or the deadline was exceeded: %v", ctx.Err())
```

Comment:
nit: above it's createCtr, here it's ctrCreate

I'm not a huge fan of abbreviating ctr here, but I understand we don't want to be logging unnecessarily.

Do we really need the function names? if so, then can you make them uniform?

Reply:
having function names will make it easier to track these.

Reply:
yeah, it is meant to be uniform - mistyped here. Quite a few of the functions in defer are the same for sandbox_run and container_create so I prefer to point out where the defer was occurring for easy debugging. I can take out the "defer" from the log, but would like to keep the function names.

Reply:
Fixed